webpack.config.js

```diff
@@ -22,53 +74,155 @@ const webpackConfigLoaders = [
   {
     test: /\.scss$/,
     loaders: [ 'style-loader', 'css-loader', 'sass-loader' ]
+  },
+
+  /**
+   * Raw loader support for *.css files
+   * Returns file content as string
+   *
+   * See: https://github.com/webpack/raw-loader
+   */
+  {
+    test: /\.css$/,
+    loaders: [ 'style-loader', 'css-loader' ]
+  },
+
+  /**
+   * Raw loader support for *.html
+   * Returns file content as string
+   *
+   * See: https://github.com/webpack/raw-loader
+   */
+  {
+    test: /\.html$/,
+    loader: 'raw-loader',
+    exclude: [ path.resolve( root, 'index.html' ) ]
+  },
+
+  /*
+   * Json loader support for *.json files.
+   *
+   * See: https://github.com/webpack/json-loader
+   */
+  {
+    test: /\.json$/,
+    loader: 'json-loader'
   }
 
 ];
 
 const webpackConfigPlugins = [
 
-  new HtmlWebpackPlugin({
+  new HtmlWebpackPlugin( {
     title: 'Tombaugh Regio',
-    template: 'src/index.html',
+    template: path.resolve( root, 'index.html' ),
     env: ENV,
     host: '0.0.0.0',
+    // port is determined from npm config
+    // which is set in package.json
+    // "config": {
+    //    "port": "9000"
+    // }
     port: process.env.npm_package_config_port,
```

Comment:
actually this is not needed at all for serving dev-server. check latest commit additions